elasticdl/python/ps/parameter_server.py

```diff
@@ -42,6 +46,26 @@ def __init__(self, args):
         self.master_name = get_master_pod_name(args.job_name)
         self.namespace = args.namespace
         self._init_checkpoint_service(args)
+        self._restore_params_from_checkpoint(args.checkpoint_dir_for_init)
+
+    def _restore_params_from_checkpoint(self, checkpoint_dir_for_init):
+        """Restore parameters from a checkpint directory for the PS instance
+        """
+        if checkpoint_dir_for_init is None:
+            self.logger.info("checkpoint directory for init is None")
+            return
+
+        if not check_checkpoint_valid(checkpoint_dir_for_init):
+            raise ValueError("Invalid checkpoint directory")
+
+        self.parameters = restore_model_params_from_checkpoint(
+            checkpoint_dir_for_init, self.ps_id, self.num_ps_pods
+        )
+        self.parameters.init_status = True
+        self.logger.info(
+            "The version of restored parameters is %s"
```

Comment:
parameters.version is a string type?

Reply:
The parameters.version type is int.
https://github.com/sql-machine-learning/elasticdl/blob/c1bcb4562a285606523f5dc51672197f2986b740/elasticdl/python/ps/parameters.py#L38

The version in model message of grpc is also int.
https://github.com/sql-machine-learning/elasticdl/blob/c1bcb4562a285606523f5dc51672197f2986b740/elasticdl/proto/elasticdl.proto#L82-L86

Reply:
Yes, I'm wondering why we use "The version of restored parameters is %s"?

Reply:
"%d" ?